notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>What's the difference between a "conditional expression" and a "conditional statement"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>This was to define the difference between an conditional expression (&gt;,...) and a conditional function (isin, notna,...). This is not very clear and provides no added value. I changed to your suggestion earlier: 'single column label, a list of column labels, a slice of labels, a conditional expression, or a colon.'</p>